scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
```

Comment:
```suggestion
	const getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
```

Use `const` wherever possible. `let` signifies that you want to modify this value somewhere down the line.